<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Audio</class>
 <widget class="QWidget" name="Audio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/res/bg.jpeg);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 900 12pt &quot;Segoe UI Black&quot;;</string>
       </property>
       <property name="text">
        <string>Lecture#01</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="styleSheet">
        <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Lecture#02</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="styleSheet">
        <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Lecture#03</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="styleSheet">
        <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Lecture#04</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="styleSheet">
        <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Lecture#05</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="styleSheet">
        <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Lecture#06</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="styleSheet">
        <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Lecture#07</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_8">
       <property name="styleSheet">
        <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Lecture#08</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_9">
       <property name="styleSheet">
        <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Lecture#09</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_10">
       <property name="styleSheet">
        <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Lecture#10</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Snap ITC&quot;;
color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Speaker&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_11">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/res/bg2.jpeg);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_12">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/res/bg2.jpeg);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
